src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -97,7 +104,7 @@ class AuthScreens extends React.Component {
                 return;
```

Comment:
I know that this isn't code from this PR, so I can consider it another NAB, but I think this is not quite done the proper way. The way I would expect this to happen is to use `componentDidUpdate` and if the network prop changed to being offline, then the interval would be cleared. Then when the network prop changes to online again, the interval would be re-created.

Reply:
Hmm, not sure I fully understand your suggestion so perhaps it would be better to have this change in a separate PR

Reply:
OK, that works. I'll maybe submit something to see if my idea works.